Common/Orders/Fills/ImmediateFillModel.cs

```diff
@@ -448,7 +448,7 @@ private static bool IsExchangeOpen(Security asset)
             return true;
         }
 
-        private class Prices
+        public class Prices
```

Comment:
could this be `protected` ?

Reply:
I did this in order to test the `GetPrices` method directly in `LatestPriceFillModelTests`. If the accessibility of `Prices` anything less than public, I cannot test the `GetPrices` method because this is the type returned by `GetPrices`. I agree it's not ideal to make it a public class, but I'm not sure what else to do.